src/Commands/LaraFormsBuilderSetupCommand.php

```diff
@@ -110,30 +115,30 @@ protected function checkEnvironment(): void
     protected function runInstallations(): void
     {
         $this->installJetstream();
+        $this->installPikadayAndMoment();
+        $this->includeConfirmationModal();
         $this->publishLaraFormsBuilderConfigFile();
         $this->publishLaraFormsBuilderCSSFile();
-        $this->includeConfirmationModal();
-        $this->installPikadayAndMoment();
         $this->npmActions();
     }
 
     protected function installJetstream(): void
     {
-        if (! $this->isJetstreamInstalled) {
-            if ($this->components->confirm('This Package Requires (laravel/jetstream:^3.0 with livewire/livewire:^2.0) Do You Want To Install them ?', true)) {
+        if (! $this->option('skip-jetstream-installation') && ! $this->isJetstreamInstalled) {
+            if ($this->components->confirm('This package qequires (laravel/jetstream:^3.0 with livewire/livewire:^2.0). Do you want to install them?', true)) {
```

Comment:
This package qequires -> This package requires

Reply:
Done.
Thanks for your review Wisam.